### ISSUE TITLE
IOTA amount disappeared

### ISSUE THREAD

**Issue by DSofter** [iotaledger/wallet#503]
Hello, has there been a protocol level upgrade I've missed? I just downloaded the latest Mac wallet (because the old version is showing network error), and I used the same seed to login and my funds are gone.

Please help!! Thanks!

**Comment by DSofter** [comment]
Just to give you a better idea what's going on.

![image](https://user-images.githubusercontent.com/1687829/33202700-59737100-d0b2-11e7-83e9-0d45d55ed504.png)

You can see from the wallet that the balance is zero, but on iotasear.ch it shows the balance.

**Comment by wonkytonky** [comment]
have you tried changing node? sometimes there's some delay/ddos/not uptodate

**Comment by DSofter** [comment]
@wonkytonky Do you have any suggestions which node or list of nodes should I try? Thanks!

**Comment by DSofter** [comment]
BTW couldn't get into the slack channel and also couldn't find the register button on `forum.iota.org`

**Comment by wonkytonky** [comment]
go trough all the nodes if you have to .. it doesnt matter what node you use..

**Comment by DSofter** [comment]
@wonkytonky Thank you!

**Comment by paulhandy** [comment]
Were you able to resolve this? If so, please close the issue.

**Comment by bartolosimpson** [comment]
Yes, the problem is node sync the balance is ok, change node try to view the correct balance and close isse, thanks!

**Comment by DSofter** [comment]
@paulhandy @bartolosimpson Sorry I have tried all the nodes in the node list and none of them works.

![image](https://user-images.githubusercontent.com/1687829/33232662-db5f0136-d1be-11e7-819e-8cdc4ee85ff2.png)

**Comment by DSofter** [comment]
Can this be fixed by running a full node? I followed the instruction here https://www.reddit.com/r/Iota/comments/7fcjyb/how_to_run_full_node_in_four_simple_steps/ but it guided me to https://iotatangle.slack.com to find the neighbors, which i don't have access to.

**Comment by zinnyko** [comment]
the same happened with me, please let me know if you find a way to resolve it.

**Comment by DSofter** [comment]
@zinnyko I fixed the issue. Download the latest version of the desktop wallet from [here](https://github.com/iotaledger/wallet/releases), then click on "Receive" and then click on "ATTACH TANGLE". Then I opened Tools -> Edit Node Configuration and changed to this node:

![image](https://user-images.githubusercontent.com/1687829/33537359-02d87fa0-d870-11e7-95ce-63c42ef555b1.png)

Wait a bit and my balance showed up:

![image](https://user-images.githubusercontent.com/1687829/33537368-0e98efa0-d870-11e7-8ef4-73063bd6caa6.png)


Hope it helps! I'm going to close this issue.
